src/components/inputs/z-combobox/index.tsx

```diff
@@ -147,6 +156,83 @@ export class ZCombobox {
     this.selectedCounter = this.itemsList.filter((item) => item.checked).length;
   }
 
+  private getControlToListSize(): ListSize {
+    let size;
+    switch (this.size) {
+      case ControlSize.BIG:
+        size = ListSize.LARGE;
+        break;
+      case ControlSize.SMALL:
+        size = ListSize.SMALL;
+        break;
+      case ControlSize.X_SMALL:
+        size = ListSize.SMALL;
+        break;
+      default:
+        size = ListSize.MEDIUM;
+    }
+
+    return size;
+  }
+
+  private getItemId(item: ComboItem): string {
+    return `#combo-checkbox-${this.inputid}-${item.id}`;
```

Comment:
Questo metodo dovrebbe ritornare la string senza il selettore `#`, in modo da poterlo utilizzare anche a linea 358